pkg/controller/status/status.go

```diff
@@ -39,6 +39,8 @@ type Configurator interface {
 	Config() *config.Controller
 }
 
+// Responsible for managing the status of the operator according the status of the sources.
+// Sources come from different major parts of the codebase, representing the status of said part of the codebase.
```

Comment:
`of said part`? What does it mean? :)

Reply:
Its trying to reference the `major` parts in the previous part of the sentence :) Will try to make it clearer